administration/intro.adoc

```diff
@@ -106,39 +106,40 @@ manifest files:
 1. Stable flow
```

Comment:
Maybe rename stable to primary?  I think this isn't about stable vs unstable anymore

Reply:
Done